src/components/PageHeader/index.module.css

```diff
@@ -0,0 +1,138 @@
+.header {
```

Comment:
I used CSS modules for this react component because its built in to Astro and I don't like tailwind and didn't wanna install a whole react styling library

Sorry it's monstrous.